### DISCUSSION TITLE
Bridge Sonoff ZB

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#1071] [Hardware & Firmware]
Ho impostato correttamente il Bridge ma capita (una volta al mese?) che HA non veda i vari sensori ZB e coincide con un led azzurro che lampeggia sul Bridge, riavvio dello stesso bridge e torno tutto a posto.
Qualche idea del perchè? C'è modo di mettere un riavvio automatico una tantum?
Grazie.

**Comment by snakuzzo** [comment]
Innanzi tutto non hai detto come lo hai integrato.
Poi...se il bridge è irraggiungibile quando è in questo stato non puoi schedulare un restart e nel caso puoi piazzare a monte una presa smart.
Il riavvio schedulato comunque  sarebbe semplicemente una pezza...non è una situazione normale. Se non riuscissi a trovare il motivo dei blocchi, al posto tuo sostituirei il bridge

**Reply by nromagna** [reply]
Integrato in cloud mode... lui lo vedo sia in HA che nell'app Sonoff, ma non vedo (li vedo non in linea) i vari sensori ZB.
Avevo pensato pure io ad una presa smart, mi sembrava però bruttino (una pezza come dici tu).

**Reply by snakuzzo** [reply]
Consiglio...flashalo ed utilizzalo con ZHA o con Zigbee2Tasmota
